src/opencl_common.c

```diff
@@ -2541,8 +2541,8 @@ cl_uint get_processors_count(int sequential_id)
  * This will produce a *guessed* figure
  */
 
-		// Volta or Turing
-		else if (strstr(dname, "TITAN V") || strstr(dname, "RTX 2"))
+		// Volta, Turing, Ampere
+		else if (strstr(dname, "TITAN V") || strstr(dname, "RTX 2") || strstr(dname, "RTX 3"))
```

Comment:
We should also detect "RTX A", and in fact I think it'd be safer to make it just "RTX" instead of listing the 2, 3, A prefixes. Yes it would also detect future RTX series, but I think it's probably best to guess the same as previous known revision.

We can also do `dname[0] == 'A' && dname[1] >= '1' && dname[1] <= '9'` for the A100, etc.